corehq/apps/app_manager/models.py

```diff
@@ -2178,6 +2200,88 @@ def update_app_case_meta(self, app_case_meta):
                 meta.add_closer(self.unique_id, action.close_condition)
 
 
+class SchedulePhaseForm(IndexedSchema):
+    """
+    A reference to a form in a schedule phase.
+    """
+    form_id = FormIdProperty("modules[*].schedule_phases[*].form_id")
+
+
+class SchedulePhase(IndexedSchema):
+    """
+    SchedulePhases are attached to a module.
+    A Schedule Phase is a grouping of forms that occur within a period and share an anchor
+    A module should not have more than one SchedulePhase with the same anchor
+
+    anchor:                     Case property containing a date after which this phase becomes active
+    forms: 			The forms that are to be filled out within this phase
+    """
+    anchor = StringProperty()
+    forms = SchemaListProperty(SchedulePhaseForm)
```

Comment:
Is it not possible to just make a `FormIdListProperty` that returns a `StringListProperty`